spa/src/analytics/context.ts

```diff
@@ -0,0 +1,34 @@
+import Api from "../api";
+
+export type UserContext = {
+	tenantId: string;
+	accountId: string;
+	clientKey: string;
+};
+
+export const getUserContext = async (): Promise<UserContext | undefined> => {
+
+	try {
+
+		const jwtToken: string = await new Promise((res) => {
```

Comment:
Can we reuse the `getHeaders` from `axiosInstance`? Maybe give it a different name like `getAtlassianJWT`.

Reply:
done.